src/borg/testsuite/archiver.py

```diff
@@ -3529,6 +3529,13 @@ def test_strip_components_doesnt_leak(self):
                 res = self.cmd('extract', "--debug", self.repository_location + '::test', '--strip-components', '0')
                 self.assert_true(marker not in res)
 
+    def test_do_not_mention_archive_if_you_can_not_find_repo(self):
+        '''https://github.com/borgbackup/borg/issues/6014'''
```

Comment:
Some highlighters do not work correctly with triple-singlequoted docstrings, please use triple-doublequoted ones.
"""..."""

Reply:
Done